test/it/application-assign-user.js

```diff
@@ -1,4 +1,5 @@
 const expect = require('chai').expect;
+const faker = require('faker');
```

Comment:
To avoid importing in `faker` in each file, we should wrap the usage into a utility method inside of `utils.js`.

For example:

```javascript
// util.js
function getMockProfile() {
  return {
    firstName: faker.name.firstName(),
    lastName: faker.name.lastName(),
    email: faker.internet.email(),
    login: faker.internet.email()
  }
}
```
```javascript
// Integration tests

const user = {
  profile: util.getMockProfile(),
  credentials: {
    password: {}
  },
  ...
}